test/fixtures/for_unit_tests/stateful_set_pod_test.yml

```diff
@@ -0,0 +1,98 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  creationTimestamp: "2019-10-07T16:05:37Z"
+  generateName: test-ss-
```

Comment:
nit: should this be `test-ss`?

Reply:
good q! `generateName` is an optional prefix that's used to generate a name (in the absence of an actual `name` field), so we typically include a hyphen at the end. This is also done in the pod configs for the DaemonSet tests [(example)](https://github.com/Shopify/krane/blob/master/test/fixtures/for_unit_tests/daemon_set_pod.yml#L5)